### ISSUE TITLE
a30s

### ISSUE THREAD

**Issue by r3k2** [SamarV-121/releases#1]
Hello, I installed yesterday your rom from November for a30s samsung phone.. and have two issues.
- Cell keeps going from 4g to none and back so my conexion goes on and off every 10 secs..
- the rom says this is for a SM-N980F build but the a30s is NOT that type of samsung but a: SM-N970F

please help with issue 1. and explain issue 2 thank you 
also is a30 compatible with a30s? I see lot of new builds for a30 but latest one for a30s is from november

**Comment by SamarV-121** [comment]
a30s? wew
some guys tested that build and it didn't boot for them so stopped making new ones

**Comment by r3k2** [comment]
oh... :( well it worked for me after some hacking on TWRP and for most part works fine.. only the cell and the weird number... instead of a30s number.
is a30 rom compatible with a30s?

**Comment by SamarV-121** [comment]
it should work fine if you flash a30s kernel over it

**Comment by r3k2** [comment]
thanks, know where I can get one? I guess I install the a30 zip/img then before booting into system add the kernel zip?  Any recomendations for a room that works 100% with a30s?

**Comment by SamarV-121** [comment]
Flash a30 build then flash stock boot.img or from a30s lineage zip

**Comment by r3k2** [comment]
thanks, which is your latest ROM for a30? I found my boot.img from stock downloaded from samsung N970FXXS6DTK8_N970FOVF6DTJ2_ATL.zip

**Comment by SamarV-121** [comment]
https://samarv-121.github.io/lineageos-18.0-universal7904/

**Comment by SamarV-121** [comment]
you'll need to remove assertion from updater_script in order to flash on a30s

**Comment by r3k2** [comment]
> you'll need to remove assertion from updater_script in order to flash on a30s

what do you exactly mean? is that a variable I have to comment or a whole section?

**Comment by SamarV-121** [comment]
Remove first line from META_INF/com/google/android/updater_script

**Comment by r3k2** [comment]
Hello this is what I have done.
I got the .zip of your ROM for a30
unzip. added the stock kernel boot.img and removed that line.
I zip it. go into twrp load/flash it.
I format
then boot into system
it loads the loading screen of lineage OS but is been like that for 20m already.

**Comment by SamarV-121** [comment]
Try boot.img from a30s lineage build

**Comment by r3k2** [comment]
hmm just did that now that I readed your msg, no luck... I tried the lineage-17.1-20210114-UNOFFICIAL-treble_arm64_bvS.img.xz
by it self and does work.. but is lineage-17... should I try with this kernel into your a30 rom? or a30s rom?

**Comment by SamarV-121** [comment]
That's system image only
It doesn't contain kernel

**Comment by SamarV-121** [comment]
Join https://t.me/Exynos7904 
And give me logcat while bootloop (adb logcat > log.txt) from los18 a30 build with a30s boot.img

**Comment by r3k2** [comment]
> That's system image only
> It doesn't contain kernel

yes what I did is put yours, then I added stock kernel, since it did not boot even after many tries I put this one over system and that worked so is a mix of all. I will try this weekend what you say. and I do not use telegram is closed, I rather use signal or matrix

**Comment by fard33n** [comment]
Hello SamarV is there a build of LineageOS 19 for android 12 from you?
I saw a post in 4pda regarding it which linked to a website with your name but it won't load.
